libr/core/cmd_print.c

```diff
@@ -1036,12 +1036,15 @@ static void cmd_print_format(RCore *core, const char *_input, const ut8* block,
 		/* This make sure the structure will be printed entirely */
 		char *fmt = input + 1;
 		while (*fmt && IS_WHITECHAR (*fmt)) {
-			fmt++;
-		}
-		int size = r_print_format_struct_size (fmt, core->print, mode, 0) + 10;
-		if (size > core->blocksize) {
-			r_core_block_size (core, size);
-		}
+      fmt++;
```

Comment:
fuck github editor for not supporting tabs

Reply:
np , i will fix it :)